lib/docsplit.rb

```diff
@@ -104,6 +104,17 @@ def self.clean_text(text)
     TextCleaner.new.clean(text)
   end
 
+  def self.extract_permissions(pdfs, opts={})
+    pdfs = ensure_pdfs(pdfs)
+    permissions = InfoExtractor.new.extract(:permissions, pdfs, opts)
+
+    return if permissions.nil?
```

Comment:
A question for reviewer?

Files that are not password protected in any way do not have permissions listed.

To improve the API we could return has with all keys set to true instead of `nil`. WDYT?

For example:

```
return {"print"=>true, "copy"=>true, "change"=>true, "addNotes"=>true} if permissions.nil?
```